browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1588,9 +1588,13 @@ L.Control.JSDialogBuilder = L.Control.extend({
 	},
 
 	_editControl: function(parentContainer, data, builder, callback) {
-		var edit = L.DomUtil.create('input', 'ui-edit ' + builder.options.cssClass, parentContainer);
+		var container = L.DomUtil.createWithId('div', data.id, parentContainer);
+		L.DomUtil.addClass(container, 'editbutton');
```

Comment:
please follow similar pattern as other controls, and it's not a button.
Maybe: `ui-edit-container` will be better?